Formula/go-jira.rb

```diff
@@ -0,0 +1,31 @@
+class GoJira < Formula
+  desc "Simple jira command-line client in Go"
+  homepage "https://github.com/Netflix-Skunkworks/go-jira"
+  url "https://github.com/Netflix-Skunkworks/go-jira/archive/v1.0.12.tar.gz"
+  sha256 "7ce5d6dcc5021f2019b0107c4ad342979d18bb110188b810017b2d94ff384a33"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/gopkg.in/Netflix-Skunkworks/go-jira.v1"
+    dir.install buildpath.children
+
+    ldflags = ["-w", "-s"]
+    cd dir do
+      system "go", "build", "-o", bin/"jira", "-ldflags", ldflags.join(" "), "cmd/jira/main.go"
```

Comment:
Haven't checked but smart 💰 is on needing a `prefix.install_metafiles` after this line.

Reply:
I had that before but it didn't seem to be doing anything. happy to re-add

Reply:
If you remove the line the final build should be missing `README.md` and `LICENSE`, at least. Adding the line should ensure they get installed. If they aren't missing you can leave the line out, but they're usually missing in this format.